content/docs/studio/troubleshooting.md

```diff
@@ -308,15 +305,15 @@ Confirm that you are correctly following the
 [procedure to send live metrics and plots](/doc/studio/user-guide/projects-and-experiments/live-metrics-and-plots)
 to Iterative Studio.
 
-Also note that live metrics and plots for an experiment are displayed only if
-its parent Git commit is present in the project table. So, before you run the
-experiment, make sure that its parent commit is pushed to Git and shown in the
+Note that a live experiment is nested under the parent Git commit in the project
```

Comment:
@mvshmakov Is there anything else we need to explain here?

Reply:
As per the https://github.com/iterative/dvc.org/pull/4638#discussion_r1242248953, should we explicitly say that the user should try `git push`?

Apart from it, looks good to me!

Reply:
Added [here](https://github.com/iterative/dvc.org/pull/4638/commits/19e4cc2cadef107080637f1ae384ee590d7e49c5)